config-controller/pkg/client/client.go

```diff
@@ -0,0 +1,203 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	storage "github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/roxctl/common"
+	"github.com/stackrox/rox/roxctl/common/auth"
+	roxctlIO "github.com/stackrox/rox/roxctl/common/io"
+	"github.com/stackrox/rox/roxctl/common/logger"
+	"github.com/stackrox/rox/roxctl/common/printer"
+)
+
+//go:generate mockgen-wrapper
+type CachedPolicyClient interface {
+	List(ctx context.Context) ([]*storage.Policy, error)
+	Get(ctx context.Context, name string) (*storage.Policy, bool, error)
+	Create(ctx context.Context, policy *storage.Policy) (*storage.Policy, error)
+	Update(ctx context.Context, policy *storage.Policy) error
+	FlushCache(ctx context.Context) error
+	EnsureFresh(ctx context.Context) error
+}
+
+//go:generate mockgen-wrapper
+type PolicyClient interface {
+	List(context.Context) ([]*storage.ListPolicy, error)
```

Comment:
I changed some of the method signatures in `CachedPolicyClient` to better accommodate the use cases of the controller.  For example, `Get` takes a `name` instead of an `id`.